lib/hawkular/alerts/alerts_api.rb

```diff
@@ -348,6 +348,8 @@ def initialize(trigger_hash)
       @context = trigger_hash['context']
       @type = trigger_hash['type']
       @tags = trigger_hash['tags']
+      @firing_match = trigger_hash['firingMatch']
```

Comment:
should there be a default value for firing_match/auto_resolve_match if it's empty in trigger_hash (or a validation for illegal value) ?